help.html

```diff
@@ -23,17 +23,25 @@
     <script src="static/js/jquery-2.1.4.min.js"></script>
     <script src="python-main.js"></script>
 
+    <!-- Just for debugging purposes. Don't actually copy these 2 lines! -->
+    <!--[if lt IE 9]><script src="../../assets/js/ie8-responsive-file-warning.js"></script><![endif]-->
+    <script src="static/js/ie-emulation-modes-warning.js"></script>
+    <script src="urlparser.js"></script>
+    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
+    <!--[if lt IE 9]>
+      <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
+:q      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
+    <![endif]-->
```

Comment:
This looks like a regression too, it was removed from master and v2-beta.
This block should only contain `<script src="urlparser.js"></script>`.

Reply:
Done